jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -116,6 +116,11 @@
           >
           </v-text-field>
         </v-row>
+        <v-row v-if="counts_factor < 0">
+          <span class="v-messages v-messages__message text--secondary" style="color: red !important">
+            Counts conversion factor cannot be negative.
+          </span>
+        </v-row>

```

Comment:
can we use validation rules in the `v-text-field` instead?

```
:rules="[() => counts_factor>=0 || 'Counts conversion factor cannot be negative.']"
```

Reply:
and while we're at it, might as well disable the action button by adding another term to the already existing `:disabled` in `plugin-action-button`

Reply:
Sure. Done and added you as co-author. Thanks for the suggestions!